test/commands/rm.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const cli = require('heroku-cli-util')
+const cmd = require('../..').commands.find(c => c.topic === 'container' && c.command === 'rm')
+const expect = require('unexpected')
+const nock = require('nock')
+
+describe('container removal', () => {
+  beforeEach(() => cli.mockConsole())
+
+  it('removes one container', () => {
+    let api = nock('https://api.heroku.com:443')
+      .patch('/apps/testapp/formation/web')
+      .reply(200, {})
+
+    return cmd.run({app: 'testapp', args: ['web'], flags: {}})
+      .then(() => expect(cli.stdout, 'to be empty'))
+      .then(() => expect(cli.stderr, 'to contain', 'Removing container web for testapp... done'))
+      .then(() => api.done())
+  })
+
+  it('removes two containers', () => {
+    let api = nock('https://api.heroku.com:443')
+      .patch('/apps/testapp/formation/web')
+      .reply(200, {})
```

Comment:
I think this would usually be a 204 but I also think that it's currently a 404 unless we fixed the API issue?

Reply:
Indeed, we return a 404. We also give an error right now (when it actually works).